ledgers/xero/README.md

```diff
@@ -0,0 +1,35 @@
+# LedgerSync::Xero
```

Comment:
We need to figure out how to handle documentation and patterns for these READMEs